EMS/common-bundle/src/Common/Cluster/AggregationResult.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CommonBundle\Common\Cluster;
```

Comment:
We already have EMS\CommonBundle\Elasticsearch. At that location I would create a new folder called Sync

Sync/
 ├── Synchronizer.php
 ├── BulkRequest.php
 ├── SearchResponse.php
 ├── Hit.php
 ├── Aggregation.php
 └── Bucket.php